src/clusterfuzz/_internal/issue_management/google_issue_tracker/issue_tracker.py

```diff
@@ -514,6 +529,17 @@ def ccs(self):
       change_list.removed.extend(removed)
     return change_list
 
+  @property
+  def collaborators(self):
```

Comment:
Do we need to add this to the base class as well? i.e. https://github.com/google/clusterfuzz/blob/709fbb62ca1e2a809dcc6b197076d97d75263115/src/clusterfuzz/_internal/issue_management/issue_tracker.py#L108

Reply:
When we are done with our changes, collaborators should only be used internally in Issue, and only for Google Issue Tracker's implementation. I've gone ahead and added an update to explicitly treat _collaborators as an internal entity, so there's no confusion about the public API.